src/index.html

```diff
@@ -1,13 +1,47 @@
 <!doctype html>
-<html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+<html lang="en" class="page">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./styles/main.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <main class="product-card" data-qa="card">
+      <img src="./images/imac.jpeg" alt="elemenPicture" class="product-card__picture">
+
+      <h2 class="product-card__description-text reset-margin">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</h2>
+      <h3 class="product-card__description-code reset-margin">Код товара: 195434</h3>
+
+      <section class="product-card-rating">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+        <div class="product-card-rating__count-review">
+          Отзывов: 5
+        </div>
+      </section>
+
+      <div class="product-card__prise">
+        <h2 class="product-card__prise-text reset-margin">Цена:</h2>
+        <h1 class="product-card__prise-amount reset-margin">69 999 грн</h1>
```

Comment:
pay attention to mentor's comment
<img width="902" alt="image" src="https://user-images.githubusercontent.com/63306833/157851585-cb047658-fb56-4db2-9df5-5b1a704de6a7.png">
```suggestion
        <h3 class="product-card__prise-text reset-margin">Цена:</h3>
        <p class="product-card__prise-amount reset-margin">69 999 грн</p>
```